indigo_app/static/javascript/indigo/views/document_editor.js

```diff
@@ -670,10 +693,26 @@
         ok();
 
       } else {
-        this.sourceEditor
+        var sourceEditor = this.sourceEditor;
+        sourceEditor
           // ask the editor to returns its contents
           .saveChanges()
           .done(function() {
+            fetch(`/api/documents/${Indigo.Preloads.document.id}/activity/edits`, {
+              method: 'POST',
+              headers: {
+                "Content-Type": "application/json",
+                "X-CSRFToken": Indigo.csrfToken
+              },
+              body: JSON.stringify(sourceEditor.editTimes)
+            }).then(response => {
+              if(response.ok) {
+                sourceEditor.editTimes = [];
+              } else {
+                alert('Could not save time spent on this document');
```

Comment:
An alert is very intrusive, I suggest simply logging to the console (which sentry picks up if it reports later). Next time they save we'll try again.

Reply:
done